### ISSUE TITLE
Adding Push Notification functionality to the Application

### ISSUE THREAD

**Issue by Jassi10000** [codeforcauseorg/edu-client#212]
## Is your feature request related to a problem? Please describe.

As the application we are targetting will be serving a lot amount of people and may be able to change their lives in developing themselves throught our content  so in this case , I came up with this below feature : 

## Describe the solution you'd like

I would like to implement the feature of Push Notification to keep the User of the Application Active ,  about  :

1. New Updates to the Application
2. Progress of his/her enrolled course
3. Messages from any course- mate or associated teacher or mentor
4. Profile Updations 
5. Important Deadlines 
6. Schedule of the next class 
 
Similarly , many things can be conveyed to the user through Push Notifications
I request all the project maintainers , mentors and contributors to give their views and reviews on this feature request of mine

## Additional context

Add any other context about the feature request here.

**Comment by Jassi10000** [comment]
@Abhishek-kumar09  Can you please review this ?

**Comment by Abhishek-kumar09** [comment]
Sounds good. Nice feature to look upon.
We can use firebase push notifications or maybe anything else, it just came up to my mind right now.
Have to figure out how the feature can be implemented with capacitor/ionic

**Comment by Jassi10000** [comment]
Yeah sir @Abhishek-kumar09  Firebase was the option that was coming to my mind while thinking of this feature 
But once it's implemented , it will definitely give a boost to the application

**Comment by Jassi10000** [comment]
@Abhishek-kumar09  Can you please give your reviews on #211 also
